src/SFA.DAS.EAS.Api/Orchestrators/StatisticsOrchestrator.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Threading.Tasks;
+using MediatR;
+using SFA.DAS.EAS.Account.Api.Types;
+using SFA.DAS.EAS.Application.Queries.GetFinancialStatistics;
+using SFA.DAS.EmployerAccounts.Api.Client;
+
+namespace SFA.DAS.EAS.Account.Api.Orchestrators
+{
+    public class StatisticsOrchestrator
```

Comment:
In all of the code we have, this is the first good usage of the orchestrator pattern that i've seen! 👍

Reply:
Aw shucks, thanks 👍